.github/workflows/python-package.yml

```
@@ -22,14 +22,8 @@ jobs:
     strategy:
       matrix:
         python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
-        TF: [1.13.1, 1.14, 1.15, 2.0, 2.1, 2.2, 2.*]
+        TF: [1.13.1, 1.14, 1.15, 2.0, 2.1, 2.2]
```

```suggestion
        TF: [1.13.1, 1.14, 1.15, 2.0, 2.1, 2.2, 2.*]
```

Then it's good to merge -- I'd just like to keep the wildcard in.